api/NavigatorID.json

```diff
@@ -196,44 +196,44 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/NavigatorID/platform",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "1"
             },
             "chrome_android": {
-              "version_added": true
+              "version_added": "18"
             },
             "edge": {
               "version_added": "12"
             },
             "firefox": {
-              "version_added": true,
+              "version_added": "1",
               "notes": [
-                "Previously, <code>platform</code> would report running on a 32-bit CPU if running the 32-bit version of Firefox on a 64-bit system. This was corrected in Firefox 69.",
+                "Prior to Firefox 69, <code>platform</code> would report running on a 32-bit CPU if running the 32-bit version of Firefox on a 64-bit system.",
```

Comment:
Drive by note fixes are OK, but doing them separately would be good, since I'm not in touch with all considerations for notes.